python_modules/dagster/dagster/_grpc/server.py

```diff
@@ -239,13 +242,14 @@ def __init__(
                 [f"{k}={v}" for k, v in loadable_target_origin._asdict().items() if v is not None]
             ),
         ):
-            loadable_targets = get_loadable_targets(
-                loadable_target_origin.python_file,
-                loadable_target_origin.module_name,
-                loadable_target_origin.package_name,
-                loadable_target_origin.working_directory,
-                loadable_target_origin.attribute,
-            )
+            with enter_loadable_target_origin_load_context(loadable_target_origin):
```

Comment:
is this sufficient scoping? I am surprised we dont hold this open all the way through `load_all_definitions` below  - assuming you want to be able to use this information at the individual definition granularity